app/assets/javascripts/admin/addon/components/admin-config-areas/color-palette.gjs

```diff
@@ -107,6 +115,47 @@ export default class AdminConfigAreasColorPalette extends Component {
     this.hasUnsavedChanges = true;
   }
 
+  async applyColorChangesIfPossible() {
+    const id = this.args.colorPalette.id;
+
+    if (!id) {
+      return;
+    }
+
+    const tags = document.querySelectorAll(`link[data-scheme-id="${id}"]`);
+
+    if (tags.length === 0) {
+      return;
+    }
+
+    let darkTag;
+    let lightTag;
+    for (const tag of tags) {
+      if (tag.classList.contains("dark-scheme")) {
+        darkTag = tag;
+      } else if (tag.classList.contains("light-scheme")) {
+        lightTag = tag;
+      }
+    }
+
+    try {
+      const data = await ajax(`/color-scheme-stylesheet/${id}.json`, {
+        data: {
+          request_dark: !!darkTag,
+        },
+      });
+      if (data?.new_href && lightTag) {
+        lightTag.href = data.new_href;
+      }
+      if (data?.new_dark_href && darkTag) {
+        darkTag.href = data.new_dark_href;
+      }
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(`Failed to apply changes to color palette ${id}`, error);
```

Comment:
Is there a purpose in logging an error to the console here? I'm wondering how we expect to handle these errors that are logged. Should we also log via ember as well? `import { error } from "@ember/debug";`

Reply:
I think leaving some kind of trail in the console can be helpful when debugging. At this point the color palette changes will have been saved successfully, but we couldn't apply the changes live for the admin making the changes, which I don't think is worth alerting in the UI and potentially cause confusion about whether the changes have been saved or not.